src/components/CreatePostInput/index.js

```diff
@@ -162,9 +149,9 @@ const CreatePostInput = ({
       resetTopicSearch();
       setPositionKeyboard('never');
       cancelAllRequest();
-      const removeCharacterAfterSpace = textSearch.split(' ')[0];
-      const newTopics = joinTopicIntoTopicList(removeCharacterAfterSpace, topics);
-      setTopics(newTopics);
+      // const removeCharacterAfterSpace = textSearch.split(' ')[0];
+      // const newTopics = joinTopicIntoTopicList(removeCharacterAfterSpace, topics);
+      // setTopics(newTopics);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code related to updating topics has been commented out. If these lines are not necessary anymore, consider removing them to improve readability.
```diff
-       // const removeCharacterAfterSpace = textSearch.split(' ')[0];
-       // const newTopics = joinTopicIntoTopicList(removeCharacterAfterSpace, topics);
-       // setTopics(newTopics);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->